lib/paypal_express/api.rb

```diff
@@ -353,6 +353,12 @@ def initiate_express_checkout(kb_account_id, amount, currency, properties, conte
         options[:return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :return_url)
         options[:cancel_return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :cancel_return_url)
 
+        properties_hash[:max_amount] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :max_amount)
```

Comment:
Nit: `properties_hash` doesn't need to be updated, we could simply have a local variable instead.